cl/settings/project/microservices.py

```diff
@@ -83,15 +90,15 @@
         "timeout": INCEPTION_TIMEOUT,
     },
     "inception-cpu-batch": {
-        "url": f"{INCEPTION_CPU_HOST}/api/v1/embed/batch",
-        "timeout": INCEPTION_TIMEOUT,
+        "url": f"{INCEPTION_CPU_EMBEDDINGS_HOST}/api/v1/embed/batch",
+        "timeout": INCEPTION_TIMEOUT * INCEPTION_EMBEDDING_TIMEOUT_MULTIPLIER,
     },
     "inception-query": {
         "url": f"{INCEPTION_CPU_HOST}/api/v1/embed/query",
         "timeout": INCEPTION_TIMEOUT,
     },
     "inception-text": {
-        "url": f"{INCEPTION_CPU_HOST}/api/v1/embed/text",
-        "timeout": INCEPTION_TIMEOUT,
+        "url": f"{INCEPTION_CPU_EMBEDDINGS_HOST}/api/v1/embed/text",
+        "timeout": INCEPTION_TIMEOUT * INCEPTION_EMBEDDING_TIMEOUT_MULTIPLIER,
```

Comment:
What if we use a separate multiplier for `inception-text`? We’d expect the timeout for `inception-cpu-batch` to be much larger right?, so having such a big timeout for computing single embeddings might make it harder to detect issues early.

Reply:
> What if we use a separate multiplier for inception-text? 

Sure! I'll refine the settings.